### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#259] [文档评论区]
# about

Zotero 非官方中文维护小组

https://zotero-chinese.com/about

**Comment by zhuyuhua23** [comment]
在word正文中引用文献时，引用处如何改为蓝色？有没有方法？谢谢。

**Reply by Syize** [reply]
如果你会使用Python的话，可以参考一下[我的Python脚本](https://github.com/Syize/link-zotero-citation-bibliography/blob/master/link-zotero-citation-bibliography.py)中修改颜色的代码

**Reply by Syize** [reply]
在[第209行](https://github.com/Syize/link-zotero-citation-bibliography/blob/master/link-zotero-citation-bibliography.py#L209)

**Comment by miaomiaomiaoli** [comment]
请问在翻译添加至笔记的时候，如何设置原文和翻译之间空一行，而不是紧密联系在一起？

**Comment by YANGYI1562** [comment]
zotero7文献导航窗口，文献预览，现在只能翻页，可不可以增加鼠标悬浮预览时对滚轮的响应 ，这样预览体验感更佳

**Comment by LuanJiaqi** [comment]
请问目前的zotero是否有支持语料库和单词库功能，或者有没有实现类似功能的插件呢？急需这方面的功能支持，感谢！

**Comment by WangYF1989** [comment]
一直有一个bug，虽然不影响使用，但感觉很别扭。插入参考文献后，zotero会自动生成一个与word文档同名的搜索分类，但分类的条目数不正确。我一共导入了没几篇文献，条目数却显示2000多。更新到zotero8了，这个bug还在。有没有办法解决？
